packages/p2p/src/components/buy-sell/buy-sell-header.jsx

```diff
@@ -52,6 +52,7 @@ const BuySellHeader = ({ is_visible, table_type, setTableType }) => {
             className={classNames('buy-sell__header', {
                 'buy-sell__header-position-static': !!buy_sell_store.api_error_message,
             })}
+            data-testid='headerContainer'
```

Comment:
Same question about naming convention, snake or camel?

Reply:
sure, done, thx for notice!